hack/update-gofmt.sh

```diff
@@ -17,18 +18,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-find_files() {
-  find . -not \( \
-      \( \
-        -wholename '*/vendor/*' \
-      \) -prune \
-    \) -name '*.go'
-}
-
-GOFMT="gofmt -s"
-bad_files=$(find_files | xargs $GOFMT -l)
-if [[ -n "${bad_files}" ]]; then
-  echo "!!! '$GOFMT' needs to be run on the following files: "
-  echo "${bad_files}"
-  exit 1
+cmd="bazel run //:gofmt --"
```

Comment:
the scripts in repo-infra are intended(?) to be used by other k8s repos, and I'm not sure how those repos feel about bazel.

Reply:
Exactly zero people do this

https://github.com/search?q=org%3Akubernetes+update-gofmt.sh&type=Code